content/influxdb/v2.0/reference/config-options.md

```diff
@@ -2249,8 +2249,12 @@ storage-wal-fsync-delay = "0s"
 ### store
 Specifies the data store for REST resources.
 
-**Options:** `bolt`, `memory`  
-**Default:** `bolt`  
+**Options:** `disk`, `memory`  
+**Default:** `disk`  
+
+{{% note %}}
+For backwards compatibility, this flag also acceptss `bolt` as a value.
```

Comment:
Thoughts on adding information about `bolt-path` and `sqlite-path` when using the `disk` option?

> When using `disk`, REST resources are stored on disk using the [bolt-path](#bolt-path) and [sqlite-path](#sqlite-path).

Reply:
Makes sense to me!